test/blackbox-tests/test-cases/trace-file.t/run.t

```diff
@@ -16,17 +16,7 @@ This captures the commands that are being run:
 
 As well as data about the garbage collector:
 
-  $ <trace.json grep '"C"' | cut -c 2- | sed -E 's/ [0-9]+/ .../g' | sort -u
-  {"name": "compactions", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ...}}
-  {"name": "evaluated-rules", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ...}}
-  {"name": "fds", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ...}}
-  {"name": "free_words", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ...}}
-  {"name": "heap_words", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ...}}
-  {"name": "live_words", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ...}}
-  {"name": "major_collections", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ...}}
-  {"name": "major_words", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ....00}}
-  {"name": "minor_collections", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ...}}
-  {"name": "minor_words", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ....00}}
-  {"name": "promoted_words", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ....00}}
-  {"name": "stack_size", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ...}}
-  {"name": "top_heap_words", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value": ...}}
+  $ <trace.json grep '"C"' | cut -c 2- | sed -E 's/([^0-9])[0-9]+/\1.../g' | sort -u
+  {"name": "evaluated-rules", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value":...}}
+  {"name": "fds", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"value":...}}
+  {"name": "gc", "pid": ..., "tid": ..., "ph": "C", "ts": ..., "args": {"live_words":...,"free_words":...,"stack_size":...,"heap_words":...,"top_heap_words":...,"minor_words":....,"major_words":....,"promoted_words":....,"compactions":...,"major_collections":...,"minor_collections":...}}
```

Comment:
It's nice that we can but different series in a counter. I don't think that was supported when I first added that.